src/main/sensors/gyro.h

```diff
@@ -70,6 +70,7 @@ typedef struct gyroConfig_s {
     uint16_t gyro_soft_notch_hz_2;
     uint16_t gyro_soft_notch_cutoff_2;
     gyroOverflowCheck_e checkOverflow;
+    int16_t  gyro_offset;
```

Comment:
This will break the contract of the parameter group. Please add at the end.

Reply:
ok.